monkey/infection_monkey/network_scanning/scan_target_generator.py

```diff
@@ -39,7 +39,7 @@ def compile_scan_target_list(
 
 
 def _remove_redundant_targets(targets: List[NetworkAddress]) -> List[NetworkAddress]:
-    reverse_dns = {}
+    reverse_dns: Dict[Any, Any] = {}
```

Comment:
What's the point of this type-hint? Also, did we start typehinting local vars?

Reply:
I think mypy complains if you declare an empty dict.

Reply:
Then a mypy ignore comment should be done instead of a useless typehint. Not very important though

Reply:
We should use better types than `Any, Any`.

Reply:
+1 on more specific types